src/govuk/macros/i18n.unit.test.mjs

```diff
@@ -0,0 +1,23 @@
+import { callMacro } from '../../../lib/jest-helpers'
+
+describe('i18n.njk', () => {
+  describe('govukPluralisedI18nAttributes', () => {
+    function callMacroUnderTest (...args) {
+      return callMacro('govukPluralisedI18nAttributes', 'macros/i18n.njk', ...args)
+    }
+
+    it('renders a single plural type', () => {
+      const attributes = callMacroUnderTest(['translation-key', { other: 'You have %{count} characters remaining.' }])
+      // Note the starting space so we ensure it doesn't stick to possible other previous attributes
+      expect(attributes).toEqual(' data-i18n.translation-key.other="You have %{count} characters remaining."')
+    })
+    it('renders multiple plural types', () => {
```

Comment:
This is really picky, and definitely a personal preference thing, but I find it considerably easier to read blocks of tests if there's an empty new line between each test.

Reply:
Yep, I've had a similar conversation with Colin about this on Slack.

Aware ideally this sort of thing wouldn't be coming up in code review – lots of stuff we can improve / automate here.